types/porep_proof_partitions.go

```diff
@@ -1,8 +1,11 @@
 package types
 
+// PoRepProofPartitions TODO(laser)
```

Comment:
PoRepProofPartitions represents the number of partitions used when creating a PoRep proof, and impacts the size of the proof.